pkg/api/v1/types.go

```diff
@@ -1475,6 +1475,27 @@ type PodLogOptions struct {
 	Previous bool `json:"previous,omitempty" description:"return previous terminated container logs; defaults to false"`
 }
 
+// PodAttachOptions is the query options to a Pod's remote attach call
+// TODO: merge w/ PodExecOptions below for stdin, stdout, etc
```

Comment:
The duplication is very unfortunate. This is the exact same feature except you don't run a command.